packages/react/src/listbox-item-base/ListboxItemBase.css.ts

```diff
@@ -16,7 +17,8 @@ export const item = recipe({
       flexDirection: "row",
       fontSize: "md",
       gap: "xs",
-      p: "xs",
+      px: "xs",
+      py: "10",
```

Comment:
should we use `minHeight: theme.size.lg` instead here? what is the menu item supposed to look like with 2 lines of text?

Reply:
addressed in #537 with `line-height`